src/Caller.cpp

```diff
@@ -980,45 +980,20 @@ void Caller::postToAnalytics(const std::string& appName) {
   }
 }
 
-/**
- * Executes the flow of the percolator process:
- * 1. reads in the input file
- * 2. trains the SVM
- * 3. calculate PSM probabilities
- * 4. (optional) calculate peptide probabilities
- * 5. (optional) calculate protein probabilities
- */
-int Caller::run() {
-  time_t startTime;
-  time(&startTime);
-  clock_t startClock = clock();
-  if (VERB > 0) {
-    std::cerr << extendedGreeter(startTime);
-  }
-  
-  std::string appName = "percolator";
-  postToAnalytics(appName);
 
-#ifdef _OPENMP
-  omp_set_num_threads(std::min((unsigned int)omp_get_max_threads(), numThreads_));
-#endif
-
-  int success = 0;
-  std::ifstream fileStream;
+std::istream& Caller::getDataInStream(std::ifstream& fileStream){
   if (!readStdIn_) {
     if (!tabInput_) fileStream.exceptions(ifstream::badbit | ifstream::failbit);
     fileStream.open(inputFN_.c_str(), ios::in);
   } else if (maxPSMs_ > 0u) {
     maxPSMs_ = 0u;
-    std::cerr << "Warning: cannot use subset-max-train (-N flag) when reading "
-              << "from stdin, training on all data instead." << std::endl;
+    std::cerr << "Warning: cannot use subset-max-train (-N flag) when reading from stdin, training on all data instead." << std::endl;
```

Comment:
Some of us have thinner terminals than others -- it's more friendly to avoid long lines when they can be easily broken up, as the original version was.

Reply:
reverted back